test.py

```diff
@@ -2,9 +2,10 @@
 
 
 ukulele = Tuning('GCEA')
-charango = Tuning('GCEAE')
+charango = Tuning('G4C5E4A4E5')
```

Comment:
not true: the middle string is octave-doubled. For now, please leave it as it was.